cmds/clients/contestcli-http/main.go

```diff
@@ -77,26 +81,56 @@ func run(verb string) error {
 			return fmt.Errorf("failed to parse job descriptor: %v", err)
 		}
 		params.Add("jobDesc", string(jobDesc))
+		resp, err := request(verb, params)
+		if err != nil {
+			return err
+		}
+		fmt.Println(resp)
+		// if wait flag set, keep polling for status till job is completed
+		if *flagWait {
+			fmt.Fprintf(os.Stderr, "\nWaiting for job to complete...")
+			var parsedResp map[string]interface{}
+			if err := json.Unmarshal([]byte(resp), &parsedResp); err != nil {
+				return fmt.Errorf("cannot decode json respnse: %v", err)
+			}
+			jobID := parsedResp["Data"].(map[string]interface{})["JobID"]
+			params.Set("jobID", fmt.Sprintf("%v", jobID))
```

Comment:
nit: use `strconv.Itoa` instead of `fmt.Sprintf`

Reply:
Done